rfcs/000-neptune-support.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: Amazon Neptune Databuilder support
+- Start Date: 2020-11-10
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
```

Comment:
tiny nit, I think we should update this?

Reply:
Yep